docs/source/general/predictions.rst

```diff
@@ -36,45 +37,19 @@ Predict on a csv file
 
 .. code-block:: python
 
-    from flash import download_data
-    from flash.tabular import TabularClassifier
-
-    # 1. Download the data
-    download_data("https://pl-flash-data.s3.amazonaws.com/titanic.zip", 'data/')
-
-    # 2. Load the model from a checkpoint
-    model = TabularClassifier.load_from_checkpoint(
-        "https://flash-weights.s3.amazonaws.com/tabular_classification_model.pt"
-    )
-
-    # 3. Generate predictions from a csv file! Who would survive?
-    predictions = model.predict("data/titanic/titanic.csv")
-    print(predictions)
-
-
-Distributed predictions
-=======================
-
-For more advanced options like distributed inference, you need to use the :func:`~flash.core.trainer.Trainer.predict` method.
-
-.. code-block:: python
-
-    from flash import Trainer
-    from flash import download_data
-    from flash.vision import ImageClassificationData, ImageClassifier
-
-
-    # 1. Download the data set
-    download_data("https://pl-flash-data.s3.amazonaws.com/hymenoptera_data.zip", 'data/')
-
-    # 2. Load the model from a checkpoint
-    model = ImageClassifier.load_from_checkpoint("https://flash-weights.s3.amazonaws.com/image_classification_model.pt")
+	from flash.core.data import download_data
```

Comment:
does this formating make any difference?

Reply:
cleaned it